src/main/java/com/waytta/HttpCallable.java

```diff
@@ -17,22 +17,24 @@
 import net.sf.json.JSONSerializer;
 
 
-class HttpCallable extends MasterToSlaveCallable<JSONObject, IOException> {
+class HttpCallable extends MasterToSlaveCallable<String, IOException> {
     private static final long serialVersionUID = 1L;
```

Comment:
Unnecessary for master → agent remoting, since they will always have the same version of the class file. The `serialVersionUID` is more to get FindBugs to shut up than anything else.